plugins/carlabase/carla.h

```diff
@@ -96,16 +114,33 @@ class CarlaInstrumentView : public InstrumentView
 private slots:
     void toggleUI(bool);
     void uiClosed();
+    void refreshButtons();
+    void onRefreshButton();
 
 private:
     virtual void modelChanged();
     virtual void timerEvent(QTimerEvent*);
 
+    void addKnob(FloatModel& knobModel);
+	void clearKnobs();
```

Comment:
Our coding convention says you should use tabs, but most part of this file uses spaces. I don't know which one will be better, but please don't mix them like this. You can change indentation after some discussions.